docs/create_tasks_table.py

```diff
@@ -2,8 +2,16 @@
 
 import re
```

Comment:
@isaac-chung just FYI: this doesn't seem to be used anywhere in the code

Reply:
@isaac-chung if this is the case, we can add to the ruff CI that it should just remove unused imports

Reply:
It's already part of the checks:
```yaml
[tool.ruff.lint]
select = ["F", "I", "E", "D"]
```

Note that that line is not part of the changes, i.e. not highlighted.

Reply:
Sorry about that, my bad. It is indeed used within the other functions that are not highlighted.